src/context-selection/use-context-selection/use-context-selection.js

```diff
@@ -15,16 +16,30 @@ export const PARAMS_SCHEMA = {
     sectionFilter: StringParam,
 }
 
+const useCustomQueryParam = (name, schema) => {
+    const [paramValue, setParamValue] = useQueryParam(name, schema)
+    const isValidSelection = useIsValidSelection()
```

Comment:
How does it know whether it's a valid selection before the change has been pushed to `use-query-param`.
So for example a data set has a default cat combo, when the data set id and the org unit id is already present and the user selects a period, when the logic reaches this point, the `useIsValidSelection` hook doesn't know about the period id yet, does it?